homeassistant/components/roborock/device.py

```diff
@@ -40,6 +40,11 @@ async def send(
 
         return response
 
+    @property
+    def api(self) -> RoborockLocalClient:
+        """Access the api client."""
+        return self._api
```

Comment:
This is not covered by tests, and it doesn't seem to be accessed by the `time` platform